#!/usr/bin/env python
PACKAGE = "selfie_park"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("idle_time", double_t, 0, "idle_time", 2.0, 1.0, 3.0)
gen.add("max_turn", double_t, 0, "max_turn", 0.8, 0.2, 1.2)
gen.add("minimal_start_parking_x", double_t, 0, "minimal_start_parking_x", -0.16, -0.2, 0.5)
gen.add("parking_speed", double_t, 0, "parking_speed", 0.4, 0.1, 1.0)
gen.add("angle_coeff", double_t, 0, "angle_coeff", 0.5, 0.0, 1.0)
gen.add("iter_distance", double_t, 0, "iter_distance", 0.2, 0.0, 1.0)

exit(gen.generate(PACKAGE, "park_server", "ParkServer"))
